objects/5a95bc/script.lua

```diff
@@ -7,37 +7,37 @@ postSetupComplete = false
 
 posMap = {
     { -- 1 player
-        Vector{-2.46, 1.08, 10.34},
+        Vector(2.54, 1.08, 10.34),
     },
     { -- 2 player
-        Vector{-2.46, 1.08, 10.34},
-        Vector{15.38, 1.08, 9.96},
+        Vector(20.38, 1.08, 9.96),
+        Vector(2.54, 1.08, 10.34),
     },
     { -- 3 player
-        Vector{-2.46, 1.08, 10.34},
-        Vector{15.38, 1.08, 9.96},
-        Vector{33.22, 1.08, 9.58},
+        Vector(20.38, 1.08, 9.96),
+        Vector(2.54, 1.08, 10.34),
+        Vector(38.22, 1.08, 9.58),
     },
     { -- 4 player
-        Vector{-2.46, 1.08, 10.34},
-        Vector{15.38, 1.08, 9.96},
-        Vector{33.22, 1.08, 9.58},
-        Vector{-20.30, 1.08, 10.72},
+        Vector(-15.30, 1.08, 10.72),
+        Vector(20.38, 1.08, 9.96),
+        Vector(2.54, 1.08, 10.34),
+        Vector(38.22, 1.08, 9.58),
     },
     { -- 5 player
-        Vector{-2.46, 1.08, 10.34},
-        Vector{15.38, 1.08, 9.96},
-        Vector{33.22, 1.08, 9.58},
-        Vector{-20.30, 1.08, 10.72},
-        Vector{51.06, 1.08, 9.20},
+        Vector(38.22, 1.08, 9.58),
+        Vector(-15.30, 1.08, 10.72),
+        Vector(56.06, 1.08, 9.20),
+        Vector(2.54, 1.08, 10.34),
+        Vector(20.38, 1.08, 9.96),
     },
     { -- 6 player
-        Vector{-2.46, 1.08, 10.34},
-        Vector{15.38, 1.08, 9.96},
-        Vector{33.22, 1.08, 9.58},
-        Vector{-20.30, 1.08, 10.72},
-        Vector{51.06, 1.08, 9.20},
-        Vector{68.90, 1.08, 8.82},
+        Vector(73.90, 1.08, 8.82),
```

Comment:
to make things a bigger easier to follow could you order these list in insertion order? i.e. each time the new coordinate is added to the bottom of the list?